app/javascript/mastodon/components/avatar.tsx

```diff
@@ -1,7 +1,7 @@
 import classNames from 'classnames';
 
 import { useHovering } from '../../hooks/useHovering';
-import type { Account } from '../../types/resources';
+import type { Account } from '../../types/account';
```

Comment:
some of these changes have been done (or variations on them) in main -- rebase might remove some